src/main/java/game/Display.java

```diff
@@ -443,7 +522,7 @@ public static void lives(final int lives, final GraphicsContext gc) {
 		}
 		gc.setStroke(Color.WHITE);
 		gc.setLineWidth(1);
-		final StringBuilder outputBuffer = new StringBuilder(lives);
+		StringBuffer outputBuffer = new StringBuffer(1);
```

Comment:
you should a StringBuilder over a StringBuffer when you do not need synchronization.